adoc/Private-Registry-Airgap.adoc

```diff
@@ -0,0 +1,247 @@
+== Deployment In Air Gap Environment 
+
+=== Introduction
+
+K3s is a lightweight Kubernetes distribution from Rancher that can be
+deployed on a single node. 
+
+=== Use Cases
+
+Testing:: The customer wants to test SUSE Private Registry, but they do not
+have a Kubernetes cluster prepared.
+Single-node setup:: A production setup for customers that want to use SUSE
+Private Registry, but do not want to deploy CaaSP, Racher's RKE or any other
+Kubernetes cluster.
+Air-gapped environments:: K3s can be deployed without access to the Internet.
+With the SUSE Private Registry running on such a cluster, it could for example
+serve as an image registry for other Kubernetes clusters in the customer's
+network.
+
+=== Requirements
+
+You should have Helm installed on the machine where you want to install k3s.
+
+Make sure that you have enough disk space. Rancher does not mention any specific
+requirements for disk size, so prepare at least a few gigabytes. It is possible
+to put the `/var/lib/rancher` directory on any partition or disk where you know
+that you have enough space. In case space starts to run out, you might encounter
+https://github.com/rancher/k3s/issues/1552[k3s issue #1552].
+
+=== Deployment
+
+. Deploy k3s into an air-gapped environment by following
+https://rancher.com/docs/k3s/latest/en/installation/airgap/#manually-deploy-images-method[the upstream guide].
+
+.. You will need the k3s binary from: https://github.com/rancher/k3s/releases
++
+There is a list of images required for basic k3s functionality and for some
+optional features (such as the Traefik ingress controller) here:
+https://github.com/rancher/k3s/releases/download/v1.18.10%2Bk3s1/k3s-airgap-images-amd64.tar[k3s-airgap-images-amd64.tar]
++
+Download both these files and transfer them to your air-gapped machine.
+
+.. On the machine where k3s should run, prepare the images and install
+the binary:
++
+[source,bash]
+----
+sudo mkdir -p /var/lib/rancher/k3s/agent/images/
+sudo cp k3s-airgap-images-amd64.tar /var/lib/rancher/k3s/agent/images/
+sudo cp k3s /usr/local/bin/k3s
+sudo chown +x /usr/local/bin/k3s
+----
+
+.. Install k3s. Do not start the service yet.
```

Comment:
This actually is not a requirement, I have used INSTALL_K3S_SKIP_ENABLE during my testing, but there's no reason users should avoid starting k3s as a systemd service...

Reply:
OK, so, what should it say?